pallets/parachain-staking/Cargo.toml

```diff
@@ -3,7 +3,7 @@ name = "parachain-staking"
 authors = [ "PureStake" ]
 description = "parachain staking pallet for collator selection and reward distribution"
 edition = "2018"
-version = "2.1.1"
+version = "2.2.0"
```

Comment:
Should this be a major bump (e.g. to `3.0.0`) since some of the storage items and/or calls are changing?